workloads/files/workload-network-script-cm.yml

```diff
@@ -0,0 +1,426 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: scale-ci-workload-script
+data:
+  run.sh: |
+    #!/bin/sh
+    set -eo pipefail
+
+    log () {
+      echo "$(date -u) $1" 1>&2
+    }
+
+    # pbench Configuration
+    log "Configuring pbench for Network benchmarks"
+    mkdir -p /var/lib/pbench-agent/tools-default/
+    echo "${USER_NAME:-default}:x:$(id -u):0:${USER_NAME:-default} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    echo "" > /var/lib/pbench-agent/tools-default/oc
+    echo "workload" > /var/lib/pbench-agent/tools-default/label
+    if [[ -v ENABLE_PBENCH_AGENTS ]]; then
+      echo "" > /var/lib/pbench-agent/tools-default/disk
+      echo "" > /var/lib/pbench-agent/tools-default/iostat
+      echo "" > /var/lib/pbench-agent/tools-default/mpstat
+      echo "" > /var/lib/pbench-agent/tools-default/perf
+      echo "" > /var/lib/pbench-agent/tools-default/pidstat
+      master_nodes=`oc get nodes --show-labels -l pbench_agent=true,node-role.kubernetes.io/master= | awk '/master/ {print $1}'`
```

Comment:
I know I said we can drop the `grep` and move the functionality to `awk`, but is the '/master/' needed now you have a label selector ''node-role.kubernetes.io/master"?  Also, do you need `--show-labels`?